code/game/gamemodes/changeling/modularchangling.dm

```diff
@@ -85,13 +91,15 @@ var/list/datum/power/changeling/powerinstances = list()
 	genomecost = 1
 	verbpath = /mob/proc/changeling_mimicvoice
 
-/datum/power/changeling/extractdna
-	name = "Extract DNA"
-	desc = "We stealthily sting a target and extract the DNA from them."
-	helptext = "Will give you the DNA of your target, allowing you to transform into them. Does not count towards absorb objectives."
-	genomecost = 2
-	allowduringlesserform = 1
-	verbpath = /mob/proc/changeling_extract_dna_sting
+//   Commenting this out for now because it is redundant, but may be used later//

```

Comment:
Do not leave commented out code in the repo. If you need this for later use, then save it, or go back in time with git. Do not contribute it to the repo.